### DISCUSSION TITLE
Add HTML to editor

### DISCUSSION THREAD

**Discussion by farreledwin** [Atri-Labs/atrilabs-engine#735] [Help: Miscellaneous]
can we paste HTML into the editor? and we can modify like margin, font-size, font weight inside the editor after we paste inside that?

**Comment by cruxcode** [comment]
Hi @farreledwin, interesting ask. It's not possible to do right now but it seems like a very useful feature. Would love to know more about your use case. Something like why this need has come up would be very useful to understand on how quickly we should roll this feature.

**Reply by farreledwin** [reply]
Yes, so if i have like HTML div and inside div is
<div>
Hello world
</div>

And i paste to the editor, the visual will show and i can edit that style inside the editor @cruxcode
